src/jdk.incubator.jextract/share/classes/jdk/incubator/jextract/tool/HandleSourceFactory.java

```diff
@@ -60,21 +60,19 @@
     protected final JavaSourceBuilder builder = new JavaSourceBuilder();
     protected final TypeTranslator typeTranslator = new TypeTranslator();
     private final List<String> libraryNames;
-    private final List<String> libraryPaths;
     private final String clsName;
     private final String pkgName;
 
-    static JavaFileObject[] generateRaw(Declaration.Scoped decl, String clsName, String pkgName, List<String> libraryNames, List<String> libraryPaths) {
-        return new HandleSourceFactory(clsName, pkgName, libraryNames, libraryPaths).generate(decl);
+    static JavaFileObject[] generateRaw(Declaration.Scoped decl, String clsName, String pkgName, List<String> libraryNames) {
```

Comment:
Question: do we still need this capability? E.g. generate just the bare var handles? Should we consolidate a bit (maybe in separate issue)